docs/zookeeper.md

```diff
@@ -0,0 +1,47 @@
+# `Zookeeper` Metrics Receiver
+
+The Zookeeper receiver collects metrics from a Zookeeper instance, using the `mntr` command. The `mntr` 4 letter word command needs
+to be enabled for the receiver to be able to collect metrics.
+
+## Configuration
+
+Following the guide for [Configuring the Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/configuration#file-location), add the required elements for your Apache web server configuration.
+
+To configure a receiver for your Zookeeper metrics, specify the following fields:
+
+| Field                 | Default                 | Description                                                                            |
+|-----------------------|-------------------------|----------------------------------------------------------------------------------------|
+| `type`                | required                | Must be `zookeeper`.                                                                   |
+| `endpoint`            | `localhost:2181`        | The url exposed by zookeeper                                                           |
+| `collection_interval` | `60s`                   | A [time.Duration](https://pkg.go.dev/time#ParseDuration) value, such as `30s` or `5m`. |
+
+Example configuration.
+
+```yaml
+receivers:
+  zookeeper:
+    endpoint: "localhost:2181"
+    collection_interval: 20s
```

Comment:
The sample configuration also needs a section to enable this metrics receiver in a pipeline.

Reply:
Oh good catch I missed that. Will update.